tests/testGaussianPsfFactory.py

```diff
@@ -50,14 +50,14 @@ def testApply(self):
                             naiveSize += 1
                     else:
                         naiveSize = fixedSize
-                    for minSize in (None, max(1, naiveSize-1), naiveSize, naiveSize + 2):
+                    for minSize in (max(1, naiveSize-1), naiveSize, naiveSize + 2):
                         if minSize <= 0:
                             continue
                         factory.minSize = minSize
-                        for maxSize in (None, naiveSize-1, naiveSize):
+                        for maxSize in (naiveSize-1, naiveSize):
                             if maxSize <= 0:
                                 continue
-                            if None not in (minSize, maxSize) and maxSize < minSize:
```

Comment:
That seems to be the case, but since the remainder of the loop is skipped over with the continue, it looks like it's safe to remove the None iteration. Apparently, the code is inherited and the origin of the None iteration is lost.